packages/amplify-e2e-core/src/configure/index.ts

```diff
@@ -20,6 +20,7 @@ export const amplifyRegions = [
   'us-west-1',
   'us-west-2',
   'eu-north-1',
+  'eu-south-1',
```

Comment:
We shouldn't add this yet. We would need to enable this region for all our test accounts and studio tests would fail until studio is deployed in this region

Reply:
removed